classes/actions.php

```diff
@@ -262,8 +286,24 @@ public static function duplicate_to_course(array $modules, int $targetcourseid,
         // Let order of mods in a section be mod1, mod2, mod3, mod4, mod5. If we duplicate mod2, mod4, the order afterwards will be
         // mod1, mod2, mod3, mod4, mod5, mod2(dup), mod4(dup).
         $duplicatedmods = [];
+        $cms = [];
+        $errors = [];
         foreach ($idsincourseorder as $cmid) {
-            $duplicatedmod = massactionutils::duplicate_cm_to_course($targetmodinfo->get_course(), $sourcemodinfo->get_cm($cmid));
+            try {
+                $duplicatedmod = massactionutils::duplicate_cm_to_course($targetmodinfo->get_course(), $sourcemodinfo->get_cm($cmid));
```

Comment:
Line exceeds 132 characters;

Reply:
Thanks for your review.
I updated the code to add a break line.